js/Source/Bootstrap.Popup/Behavior.BS.Popup.js

```diff
@@ -50,19 +50,19 @@ Behavior.addGlobalFilters({
           })
         )
       );
-      popup.addEvent('destroy', function(){
-        api.cleanup(el);
-      });
```

Comment:
You can't remove this. If you remove this, closing the popup will leave any behaviors on its contents still applied.